app/invocation/invocation.tsx

```diff
@@ -62,6 +64,7 @@ export default class InvocationComponent extends React.Component<Props, State> {
     error: null,
 
     model: new InvocationModel(),
+    attempt: new Long(0),
```

Comment:
Since `model` already encapsulates the Invocation proto, could we instead add a getter to `InvocationModel` like `getAttempt()` which returns `response.invocation[0]?.attempt || 1`

Reply:
yup, sounds good.